node-it/src/test/scala/com/wavesplatform/it/Docker.scala

```diff
@@ -263,7 +263,7 @@ class Docker(
       val containerConfig = ContainerConfig
         .builder()
         .image(imageName)
-        .exposedPorts(s"$internalDebuggerPort")
+        .exposedPorts(if (Try(nodeConfig.getStringList("waves.extensions").contains("com.wavesplatform.events.BlockchainUpdates")).getOrElse(false)) "6881" else s"$internalDebuggerPort")
```

Comment:
This completely replaces port mapping when BU is enabled, which means one will not be able to debug the node inside the container. Please change this to always expose debug port, and expose BU port as needed.

Reply:
Done